features/account/customer_account/viewing_orders_history/viewing_order_recalculated_by_exchange_rate.feature

```diff
@@ -11,7 +11,7 @@ Feature: Viewing details of an order
         And the store operates on a channel named "Web"
         And that channel allows to shop using the "USD" currency
         And that channel allows to shop using the "GBP" currency with exchange rate 3.0
-        And that channel uses the "USD" currency by default
+        And that channel uses the "USD" currency as base
```

Comment:
Not exactly. Channel can operate only on one currency from the beginning. If we want to be so semantical strict, the sentence should be ` And that channel base display currency is "USD"`. Anyway, with your last argument there is no point in changing it now, because this sentence will be for 100% in one/two days.